CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# Contributing #
+## Why contributing? ##
+The original author and all users of this library are very greatful for your contribution
+to this Open Source Project. Also most employers value people active in the Open Source
+community.
+
+## The Checklist ##
+If you want to help out and contribute to this Open Source Project, please keep reading.
+
+My thought with `jadb` was to make it very light weight and easy to use as a developer.
+This means as little boilerplate code, (mostly) self-documenting public interface, and
+overall clean code.
+
+Before submitting a pull request, please go through the below checklist to verify
+your proposed change meets, or exceeds, the quality of the jadb source code.
+
+* Builds - Make sure the code builds by issuing `ant build`.
```

Comment:
Is there actually such a target in ant? Because my output is such:

```
ant build
Buildfile: /home/art/jadb/build.xml

BUILD FAILED
Target "build" does not exist in the project "jadb". 
```